src/core/components/link/ButtonLink.tsx

```diff
@@ -0,0 +1,44 @@
+import { ButtonHTMLAttributes } from 'react';
+import { SharedLinkProps } from './types';
+import { linkStyles } from './styles';
+import { linkContents } from './shared';
+
+export interface ButtonLinkProps
+	extends ButtonHTMLAttributes<HTMLButtonElement>,
+		SharedLinkProps {}
+
+/**
+ * [Storybook](https://guardian.github.io/source/?path=/docs/source-src-link-button-link--demo) •
```

Comment:
```suggestion
 * [Storybook](https://guardian.github.io/source/?path=/docs/source-src-link-button-link) •
```

this should force storybook to open the first story whatever it's called (in this not `demo` any more?)